config/enabled.yml

```diff
@@ -954,6 +954,11 @@ Style/StabbyLambdaParentheses:
   StyleGuide: '#stabby-lambda-with-args'
   Enabled: true
 
+Style/StderrPuts:
+  Description: 'Use `warn` instead of `$stderr.puts`.
```

Comment:
There's a missing ' here.

Reply:
Updated, re-run `rake generate_cops_documentation` and rebased.